tests/ui/app/reviews.js

```diff
@@ -132,7 +136,7 @@ casper.test.begin('Test review button hidden when logged in and cannot rate', {
             helpers.fake_login();
         });
 
-        helpers.waitForPageLoaded(function() {
+        waitForReviewsLoaded(function() {
```

Comment:
does this work? it seems like it would be the same issue. After waitForPageLoaded finishes, ```.app-reviews``` will already exist.